### PR TITLE
Fix Windows Jobs and add Ubuntu jobs

### PR DESCRIPTION
**Pull request by adiaaida**
The python script is case sensitive on the build type. Change debug and
release to all lower case. Add the Ubuntu jobs for debug and release

### CODE REVIEW

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

**Comment by adiaaida** [comment]
@russellhadley ptal

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -142,3 +142,38 @@ C:\\Python27\\python %WORKSPACE%\\llvm\\tools\\llilc\\test\\llilc_checkpass.py -
...
+    export CXX=$(which clang++-3.5)
+elif which clang; then
+	export CC=$(which clang)
+    export CXX=$(which clang++) 
+else
+    echo Could not find clang or clang-3.5
+    exit 1
+fi
+
+(cd coreclr && ./build.sh skipmscorlib ${configuration} && cd ..) && (cd llvm && mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=${configuration} -DWITH_CORECLR=../../coreclr/bin/Product/Linux.x64.${configuration} .. && make -j 5)''')
```
Comment (kyulee1) [inline netci.groovy]: I think directory name is still capitalized -- e.g., Linux.x64.Debug or Linux.x64.Release

Comment (adiaaida) [inline netci.groovy]: Updated to reflect how they do it in coreclr (using configuration = Debug/Release, and lowerConfigurations = debug/release)

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

**Pull request merged**